core/capabilities/remote/executable/client.go

```diff
@@ -208,10 +208,11 @@ func (c *client) sendRequest(req *request.ClientRequest) error {
 	c.lggr.Debugw("executing remote execute capability", "requestID", req.ID())
 
 	if _, ok := c.requestIDToCallerRequest[req.ID()]; ok {
-		return fmt.Errorf("request for ID %s already exists", req.ID())
+		c.lggr.Debugw("request for ID already exists", "requestID", req.ID())
```

Comment:
imo easier to read if log and return and fall thru

```
if _, exists := ....; exists {
log.Debug
return nil
}
c.foo = bar
```

Reply:
Fair. I don't have a strong opinion, so I'll change it